geomesa-fs/geomesa-fs-tools/bin/install-hadoop.sh

```diff
@@ -18,7 +18,7 @@ htrace_core_version="4.1.0-incubating"
 
 # for hadoop 2.5 and 2.6 to work we need these
 # These should match up to what the hadoop version desires
-guava_version="11.0.2"
+guava_version="%%guava.version%%"
```

Comment:
If we don't have a dependency on Guava, does the install script need to download it still?

Reply:
i think so b/c hadoop still needs it. we also do still have a dependency on guava byte comparators - they have some native code performance things that seemed useful to keep, and afaik we haven't ever hit a version issue with that particular class.